src/fluree/db/server_settings.clj

```diff
@@ -701,7 +682,7 @@
 (comment
 
 
-  environ/env
+  environ.core/env
 
   (->
-    (build-env environ/env)))
+    (build-env environ.core/env)))
```

Comment:
Same question

Reply:
We/I turned off linting against comments since the linter(s) complained about various items (e.g., unnecessary let, unused bindings, ...)  

The above code is in the comment section.  The other option would be to delete these comment lines?